Dockerfile

```diff
@@ -3,12 +3,12 @@ FROM clojure AS builder
 WORKDIR /app
 COPY ./project.clj ./
 COPY . .
-RUN lein uberjar
+RUN lein with-profile uberjar uberjar
 
 # second stage -- executable
 FROM openjdk:alpine
 WORKDIR /usr/src/app
 COPY --from=builder /app/target/kamal.jar ./target/kamal.jar
 COPY --from=builder /app/resources/ ./resources
 EXPOSE 3000
-CMD ["java", "-Xmx500m", "-Xss512k", "-jar", "target/kamal.jar"]
+CMD ["java", "-Xmx500m", "-XX:+UseG1GC", "-XX:+UseStringDeduplication", "-jar", "target/kamal.jar"]
```

Comment:
Why these flags are necessary?

Reply:
the G1 GC (garbage collector) is specially targeted to multi-threaded server programs which is precisely what we are doing. It is however designed to handle high-memory servers (> 6 GB). For the time being it is an experiment to see if it would improve things once we put lots of cities in a single server